tests/debugpy/test_evaluate.py

```diff
@@ -155,21 +160,45 @@ def code_to_debug():
             "__a_test",
             "__b_test",
             "__c_test",
-            "__a_test__",
-            "__b_test__",
-            "__c_test__",
         ]
 
+        special_vars_entry = [v for v in vars if v["name"] == "special variables"][0]
```

Comment:
I usually use single-element tuple destructuring instead of indexing for cases like these, since it also implicitly asserts that there's only one such element, and throws otherwise; i.e.:
```py
special_vars_entry, = [v for v in vars if v["name"] == "special variables"]
```